clang/test/AST/explicit-base-class-move-cntr.cpp

```diff
@@ -0,0 +1,43 @@
+// RUN: %clang_cc1 -ast-dump -fblocks %s | FileCheck -strict-whitespace %s
```

Comment:
can this test be autogenerated?

Reply:
Unfortunately, I have no idea how to generate tests for clang. This test has been manually written.

Reply:
I was thinking about utils/update_cc_test_checks.py
But this is AST test

Maybe llvm-project/clang/test/AST/gen_ast_dump_json_test.py

Reply:
@vitalybuka Thank you for the suggestion. I've regenerated the test using the tool and cut the dump to make it a bit shorter.